README.md

```diff
@@ -61,20 +60,17 @@ using Metalogger to ensure that a switch-over to a different logging implementat
 > npm install metalogger
 ```
 
-If you are planning to use syslogging via ain2 bridge, or loggly, you should
-install them manually, since they were made into optional dependencies:
+The `util` logging bridge relies on the built-in node `util`; for any other
+logging type, you must the underlying logger manually; metalogger does not
```

Comment:
A word is missing: 

> you must "install" the underlying logger manually.

Also, semicolon ";" is unnecessary. Let's just start a new sentence there.